README.template.md

```diff
@@ -70,6 +70,21 @@ echo 'export PATH="$PATH:$HOME/.bbin/bin"' >> ~/.zshrc && exec /bin/zsh
 echo 'export PATH="$PATH:$HOME/.bbin/bin"' >> ~/.bashrc && exec /bin/bash
 ```
 
+### Windows (Manual)
+
+**1. Install `bbin` CLI (and batch file, because Windows):**
+```shell
+> set BBIN_DEST=%HOMEDRIVE%%HOMEPATH%\.bbin\bin
+> mkdir %BBIN_DEST% && curl -o %BBIN_DEST%\bbin -L https://raw.githubusercontent.com/babashka/bbin/v{{version}}/bbin && curl -o %BBIN_DEST%\bbin.bat -L https://raw.githubusercontent.com/babashka/bbin/v{{version}}/bbin.bat
```

Comment:
Right now this will fail with "A subdirectory or file %BBIN_DEST% already exists." if the directory already exists. I'm going to add an `IF NOT EXIST` check to make the command idempotent.